MIGRATION.md

```diff
@@ -418,26 +422,34 @@ The new CLI commands remove the following flags:
 | -------- | --------------------------------------------------------------------------------------------- |
 | --modern | No migration needed. [All ESM code is modern in SB7](#modern-esm--ie11-support-discontinued). |
 
-#### Framework field mandatory
+#### New Framework API
 
 _Has automigration_
 
-In 6.4 we introduced a new `main.js` field called [`framework`](#mainjs-framework-field). Starting in 7.0, this field is mandatory.
-The value of the `framework` field has also changed.
+Storybook 7 introduces the concept of `frameworks`, which abstracts configuration for `renderers` (e.g. React, Vue), `builders` (e.g. Webpack, Vite) and defaults to make integrations easier. This requires quite a few changes, depending on what your project is using. **We recommend you to use the automigrations**, but in case the command fails or you'd like to do the changes manually, here's a guide:
 
-In 6.4, valid values included `@storybook/react`, `@storybook/vue`, etc.
+> Note:
+> All of the following changes can be done automatically either via `npx storybook@next upgrade --prerelease` or via the `npx storybook@next automigrate` command. It's highly recommended to use these commands, which will tell you exactly what to do.
 
-In 7.0, frameworks also specify the builder to be used. For example, The current list of frameworks include:
+##### Available framework packages
 
-- `@storybook/angular`
-- `@storybook/ember`
+In 7.0, `frameworks` combine a `renderer` and a `builder`, with the exception of a few packages that do not contain multiple builders, such as `@storybook/angular`, which only has Webpack 5 support.
+
+You have to pick which framework you want to use from the list below, which will depend on your project configuration. If you're using a framework that has multiple builders, you'll have to pick one. For example, if you're using `@storybook/react`, you'll have to pick between `@storybook/react-vite` and `@storybook/react-webpack5`. If you're using a framework that only has one builder (and therefore hasn't changed), you can just use that.
+
+Additionally, there are framework packages which are specific to meta-frameworks, like Next.js and SvelteKit. If you pick them, make sure to also see [this section]().
+
+The current list of frameworks include:
+
+- `@storybook/angular` (did not change)
+- `@storybook/ember` (did not change)
 - `@storybook/html-vite`
 - `@storybook/html-webpack5`
-- `@storybook/nextjs`
 - `@storybook/preact-vite`
 - `@storybook/preact-webpack5`
 - `@storybook/react-vite`
 - `@storybook/react-webpack5`
+- `@storybook/nextjs`
```

Comment:
why did the order change? alphabetical better?

Reply:
I made it close to react as nextjs is a react thing, so people looking for react would quickly notice it. Happy to change this!